framework/test/unit/modules/poa/commands/register_authority.spec.ts

```diff
@@ -113,6 +115,69 @@ describe('RegisterAuthority', () => {
 				.createCommandVerifyContext<RegisterAuthorityParams>(registerAuthoritySchema);
 		});
 
+		it(`should throw error when name is longer than ${MAX_LENGTH_NAME}`, () => {
+			expect(() =>
+				validator.validate(registerAuthorityCommand.schema, {
+					...registerAuthorityTransactionParams,
+					name: 'aaaaaaaaaaaaaaaaaaaaaaa',
+				}),
+			).toThrow(`Property '.name' must NOT have more than 20 characters`);
+		});
+
+		it(`should throw error when bls key shorter than ${LENGTH_BLS_KEY}`, () => {
+			expect(() =>
+				validator.validate(registerAuthorityCommand.schema, {
+					...registerAuthorityTransactionParams,
+					blsKey: utils.getRandomBytes(LENGTH_BLS_KEY - 1),
+				}),
+			).toThrow(`Property '.blsKey' minLength not satisfied`);
+		});
+
+		it(`should throw error when bls key longer than ${LENGTH_BLS_KEY}`, () => {
+			expect(() =>
+				validator.validate(registerAuthorityCommand.schema, {
+					...registerAuthorityTransactionParams,
+					blsKey: utils.getRandomBytes(LENGTH_BLS_KEY + 1),
+				}),
+			).toThrow(`Property '.blsKey' maxLength exceeded`);
+		});
+
+		it(`should throw error when proof of possession shorter than ${LENGTH_PROOF_OF_POSSESSION}`, () => {
+			expect(() =>
+				validator.validate(registerAuthorityCommand.schema, {
+					...registerAuthorityTransactionParams,
+					proofOfPossession: utils.getRandomBytes(LENGTH_PROOF_OF_POSSESSION - 1),
+				}),
+			).toThrow(`Property '.proofOfPossession' minLength not satisfied`);
+		});
+
+		it(`should throw error when proof of possession longer than ${LENGTH_PROOF_OF_POSSESSION}`, () => {
+			expect(() =>
+				validator.validate(registerAuthorityCommand.schema, {
+					...registerAuthorityTransactionParams,
+					proofOfPossession: utils.getRandomBytes(LENGTH_PROOF_OF_POSSESSION + 1),
+				}),
+			).toThrow(`Property '.proofOfPossession' maxLength exceeded`);
+		});
+
+		it(`should throw error when generator key shorter than ${LENGTH_GENERATOR_KEY}`, () => {
+			expect(() =>
+				validator.validate(registerAuthorityCommand.schema, {
+					...registerAuthorityTransactionParams,
+					generatorKey: utils.getRandomBytes(LENGTH_GENERATOR_KEY - 1),
+				}),
+			).toThrow(`Property '.generatorKey' minLength not satisfied`);
+		});
+
+		it(`should throw error when generator key longer than ${LENGTH_GENERATOR_KEY}`, () => {
+			expect(() =>
+				validator.validate(registerAuthorityCommand.schema, {
+					...registerAuthorityTransactionParams,
+					generatorKey: utils.getRandomBytes(LENGTH_GENERATOR_KEY + 1),
+				}),
+			).toThrow(`Property '.generatorKey' maxLength exceeded`);
+		});
+
```

Comment:
can we move this outside `describe('verify')` and instead create a new one `describe('verify schema')` similar to other commands